packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/updateModal.css

```diff
@@ -0,0 +1,65 @@
+.root {
+    background-color: white;
+    left: 50%;
+    top: 25%;
+    max-width: 740px;
+    opacity: 0;
+    overflow: hidden;
+    position: fixed;
+    transform: scale(1.15);
```

Comment:
Why do we scale it up when not displayed? This causes a "shrinking" appearance instead of it just sliding in. I'll try to capture it in a gif. If you change this to `scale(1)` and the open translate to `translate(-50%, 0)` it just slides in without a scaling transform. I prefer this but I also don't know what you've discussed with UX so I'd just run it by them before having it merged.

[![Image from Gyazo](https://i.gyazo.com/fda4ec35089df9ac9c3998ef72cb1b2e.gif)](https://gyazo.com/fda4ec35089df9ac9c3998ef72cb1b2e)

Reply:
I just stole it from @tjwiebell , lol. I was curious about this too actually.

Reply:
While I liked the look of it scaling _up_, I removed scaling altogether (for consistency) in https://github.com/magento/pwa-studio/pull/2280/commits/12fea240f5712df18f060be17ecdd4c22c26f783.

Reply:
![](https://i.imgflip.com/36naqm.png)